csharp-intermediate-topics/TupleAliasInCSharp/TupleAliasInCSharp/TupleAliasInCSharp.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net7.0</TargetFramework>
+    <DisableImplicitUsings>enable</DisableImplicitUsings>
+    <Nullable>enable</Nullable>
+    <LangVersion>preview</LangVersion>  
```

Comment:
With dotnet 7.0 installed on my computer I wasn't able to compile the code. And dotnet 7.0 is not a preview so I think the correct version of the framework should be 8.0 a not 7.0 and we should also point that to the readers, otherwise they will get confused, like me :)

Reply:
This PR has been updated to reflect this change.